app/VSTS/VSTS.tsx

```diff
@@ -1,39 +1,42 @@
 /// <reference path="../../office.d.ts" />
```

Comment:
you should not change the vsts file. I mean don't include it in your pull request.